fserver.install

```diff
@@ -249,3 +251,61 @@ function fserver_update_7000() {
   db_add_index('fserver_release', 'version_major', array('version_major'));
   db_add_index('fserver_release', 'project_short_name', array('project_short_name'));
 }
+
+/**
+ * Impelements hook_update_N().
+ *
+ * Add major version specific values for supported release, default major and recommended release
+ */
+function fserver_update_7001() {
+  $temp_table_name = 'temp_fserver_project';
+  // create a temporary table to store old values
+  $schema = drupal_get_schema('fserver_project', TRUE);
+  $fields = array_keys($schema['fields']);
+  db_drop_table($temp_table_name);
+  $table = db_create_table($temp_table_name, $schema);
+
+  // load old data into this table
+  $select = db_select('fserver_project', 'proj')
+    ->fields('proj')
+    ->execute();
+  $id = db_insert($temp_table_name)
+    ->fields($fields);
+  while ($record = $select->fetchAssoc()) {
```

Comment:
The only question here is can we do this all in one step or should be use batch api to make the update?